spec/controllers/search_controller_spec.rb

```diff
@@ -24,15 +30,15 @@
         it 'filter by user email' do
           get :index, search: { email: 'bar@email.com' }
 
-          expect(assigns(:matching_users)).to eq([user_three])
+          expect(assigns(:matching_users)).to include(user_three)
         end
       end
 
       context 'when have no passed email' do
         it 'sets the correct instance variables' do
           get :index, search: { email: '' }
```

Comment:
Seems like this and the earlier test for redirection are the same. Could just use these parameters in the above test to ensure it redirects.